media.css

```diff
@@ -63,7 +65,8 @@
     flex-direction: row;
     justify-content: space-between;
     align-items: center;
-    border-top-right-radius: 24px solid;
+
+    /* border-top-right-radius: 24px solid; */
```

Comment:
- [ ] According to the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) point number 12, you are supposed to remove all commented unwanted pieces of code from your code base. Having them in production code clutters the production code and is generally bad practice.